test/support/upload_multipart_controller.ex

```diff
@@ -0,0 +1,20 @@
+defmodule OpenApiSpexTest.UploadMultipartController do
+  @moduledoc tags: ["uploads"]
+
+  use Phoenix.Controller
+  use OpenApiSpex.Controller
+
+  alias OpenApiSpexTest.Schemas
+
+  plug OpenApiSpex.Plug.CastAndValidate, json_render_error_v2: true, replace_params: false
+
+  @doc request_body: {"Files", "multipart/form-data", Schemas.UploadRequest},
```

Comment:
Ideally this could be handled by setting the `encoding` on the mediaType, something like:

```
{"Files",
          %{
            "multipart/form-data" => [
              encoding: %{"files" => %OpenApiSpex.Encoding{style: "deepObject"}}
            ]
          }, Schemas.UploadRequest},
```

However it didn't seem to have any effect on the generated curl commands or request body sent from the swagger UI.